examples/problems/1200_conditional_entropy_variance.ipynb

```diff
@@ -0,0 +1,387 @@
+{
```

Comment:
<p>when having mulitple plots can you please do this with matplotlib subplot with different axes</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/79/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
I am having issues with adding colorbars to matplotlib subplots, so I replicated the subplot format using two individual paired scatter plots.